oak_functions/sdk/test_utils/src/lib.rs

```diff
@@ -180,6 +189,75 @@ where
     }
 }
 
+/// Builds the crate identified by the given name as a Linux binary, and returns the path of the
+/// resulting binary.
+fn build_rust_crate_linux(crate_name: &str) -> anyhow::Result<String> {
+    duct::cmd!(
+        "cargo",
+        "build",
+        "--target=x86_64-unknown-linux-musl",
+        "--release",
+        format!("--package={crate_name}"),
+    )
+    .dir(env!("WORKSPACE_ROOT"))
+    .run()
+    .context(format!("could not compile {crate_name}"))?;
+    Ok(format!(
+        "{}target/x86_64-unknown-linux-musl/release/{crate_name}",
+        env!("WORKSPACE_ROOT")
+    ))
+}
+
+/// Builds the crate identified by the given name as a Wasm module, and returns the path of the
+/// resulting binary.
+pub fn build_rust_crate_wasm(crate_name: &str) -> anyhow::Result<String> {
```

Comment:
Is there a difference between `build_rust_crate_wasm` and `build_rust_crate_binary` apart from the target? Should we maybe combine them if not with "build_rust_crate(name, target)"? That would get a simpler name too.

Reply:
The output files generated in the two cases have different suffixes, so I think making this a single function that just branches based on the target in order to do one of two different things would not really buy us much. But maybe in the future if we need to compile to even more targets that may make sense?